imi/src/main/java/org/motechproject/nms/imi/domain/FileAuditRecord.java

```diff
@@ -17,7 +17,10 @@
 
     @Field
     @Column(allowsNull = "false")
-    private String status;
+    private Boolean success;
+
+    @Field
+    private String error;
 
     @Field
     private Integer recordCount;
```

Comment:
It's null when the entire file fails. But should probably be >= 0 when not null.